### PR TITLE
Temporarily skip AutoMM HPO tests (lightning 1.7)

### PR DESCRIPTION
**Pull request by zhiqiangdon**
Lightning 1.7 seems not fully compatible with ray tune, which causes the AutoMM HPO tests to get stuck. Temporarily skip the AutoMM HPO tests. Need to investigate how to make ray tune or ray lightning work with lightning 1.7.  

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by yinweisu — APPROVED** [review]
LGTM! Let's wait for the CI to pass before merging in. Thanks!

**Comment by sxjscience** [comment]
@yinweisu @zhiqiangdon Let's skip the HPO test first instead of downgrading lightning. We should fix our the HPO part of AutoMM.

**Comment by sxjscience** [comment]
AutoMM + HPO is largely still experimental and require additional work. We may also wait for https://github.com/ray-project/ray_lightning/pull/196 . Meanwhile, let's find the reason for HPO test to stuck.

**Comment by yinweisu** [comment]
> @yinweisu @zhiqiangdon Let's skip the HPO test first instead of downgrading lightning. We should fix our the HPO part of AutoMM.

That's another option. We are not sure why PL 1.7 will hang with ray tune yet. My question is: is there a specific reason that we want to upgrade to PL 1.7? If it's an issue between ray tune and PL 1.7, it's likely that we won't be able to resolve it in a short time and we will lose hpo capability.

**Comment by yinweisu** [comment]
> AutoMM + HPO is largely still experimental and require additional work. We may also wait for [ray-project/ray_lightning#196](https://github.com/ray-project/ray_lightning/pull/196) . Meanwhile, let's find the reason for HPO test to stuck.

Sure. I'll try to upgrade ray to 2.0 next week to see if it helps

**Comment by sxjscience** [comment]
@yinweisu Upgrading lightning will unlock the official support of torch 1.12. We may need to find the root cause of the hanging and enable some tests again.

Does the hanging happen very often? Also, which test hanged?

**Comment by yinweisu** [comment]
> @yinweisu Upgrading lightning will unlock the official support of torch 1.12. We may need to find the root cause of the hanging and enable some tests again.
> 
> Does the hanging happen very often? Also, which test hanged?

It seems to happen quite often and @zhiqiangdon find that hpo tests are the tests that got stuck. It could be tricky to find the cause as there's no log at all. I'll try upgrading ray to 2.0 as the first step because that's something I plan to do anyway

**Comment by zhiqiangdon** [comment]
> @yinweisu Upgrading lightning will unlock the official support of torch 1.12. We may need to find the root cause of the hanging and enable some tests again.
> 
> Does the hanging happen very often? Also, which test hanged?

https://github.com/awslabs/autogluon/blob/master/multimodal/tests/unittests/test_hpo.py#L15
I saw it got stuck at the 4th hpo test.

**Comment by sxjscience** [comment]
> > @yinweisu Upgrading lightning will unlock the official support of torch 1.12. We may need to find the root cause of the hanging and enable some tests again.
> > Does the hanging happen very often? Also, which test hanged?
> 
> It seems to happen quite often and @zhiqiangdon find that hpo tests are the tests that got stuck. It could be tricky to find the cause as there's no log at all. I'll try upgrading ray to 2.0 as the first step because that's something I plan to do anyway

Yes, let's upgrade to ray 2.0. We should upgrade it anyway.

**Pull request merged**
